R/app-driver.R

```diff
@@ -130,6 +130,9 @@ NULL
 #'   Defaults to the resolved `timeout` value during the `AppDriver` initialization.
 #' @param timeout_ Amount of time to wait before giving up (milliseconds).
 #'   Defaults to the resolved `timeout` value during the `AppDriver` initialization.
+#' @param transform Optionally, a function to scrub sensitive or stochastic text
+#'        from the output. Should take a character vector of lines as input and
+#'        return a modified character vector as output.
```

Comment:
Indent seems off here

```suggestion
#' @param transform Optionally, a function to scrub sensitive or stochastic text
#'   from the output. Should take a character vector of lines as input and
#'   return a modified character vector as output.
```